pkg/loader/compose/compose.go

```diff
@@ -358,6 +358,10 @@ func (c *Compose) LoadFile(files []string) (kobject.KomposeObject, error) {
 			}
 		}
 
+		if len(serviceConfig.Port) == 0 && composeServiceConfig.Labels["kompose.service.type"] == "NodePort" || composeServiceConfig.Labels["kompose.service.type"] == "LoadBalancer" {
+			log.Fatalf("In %s service, %s defined with no ports present. Issues may occur when bringing up artifacts.", name, composeServiceConfig.Labels["kompose.service.type"])
```

Comment:
`In %s service` isn't grammatically correct.

My previous comment may work a bit better:
```
log.Fatal("%s defined in service %s with no ports present. Issues may occur when bringing up artifacts.", composeServiceConfig.Labels["kompose.service.type"], name)
```